compare-processing/results/RMA_GSE39842_annotation_df.tsv

```diff
@@ -0,0 +1,13 @@
+sample_name	dnHIF1a	amputated
```

Comment:
The _ordering_ is what is different between each of the annotation files. So the way to get around it would be to write one to file and then check if the sample names are in the same order. I think that feels slightly less straightforward to me given the way this pipeline is set up. More generally, though, I think it would be the right approach.